cypress/e2e/registration.cy.js

```diff
@@ -1,11 +1,52 @@
+/* eslint-disable max-len */
 /// <reference types='cypress' />
 
 describe('Student Registration page', () => {
-  before(() => {
-
+  beforeEach(() => {
+    // Przechodzenie do strony przed każdym testem
+    cy.visit('https://demoqa.com/automation-practice-form');
   });
 
-  it('', () => {
+  it('Wypełnij formularz bez pola "picture"', () => {
+    // Wprowadź dane do formularza
+    const firstName = 'Imie';
+    const lastName = 'Nazwisko';
+    const userEmail = 'email@example.com';
+    const userNumber = '1234567890';
+    const subjects = 'Running';
+    const currentAddress = 'Adres testowy';
```

Comment:
Please use a different randomly generated data each time you start a test. You can do it by adding a custom command 'generateUser' into the support folder or config file.